ext/date/tests/DateTime_verify.phpt

```diff
@@ -27,7 +27,7 @@ object(ReflectionClass)#%d (1) {
   string(8) "DateTime"
 }
 ..and get names of all its methods
-array(18) {
+array(19) {
```

Comment:
This test is extremely important, since reflection of internal classes is a constant SNAFU